layouts/tag.njk

```diff
@@ -0,0 +1,37 @@
+{% extends "layouts/base.njk" %}
+
+{# Create breadcrumbs manually as tag should be excluded from collections #}
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    classes: "govuk-!-display-none-print",
+    items: [{
+      href: options.parentSite.url,
+      text: options.parentSite.name
+    }, {
+      href: options.pathPrefix,
+      text: options.homeKey
+    }, {
+      href: options.pathPrefix + "tags",
+      text: "Tags"
+    }]
+  }) if options.parentSite }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    {{ appDocumentHeader({
+      title: "Posts tagged ‘" + tag + "’" | smart,
```

Comment:
Possibly this title should get set by the page using the template?

Reply:
I tried that, and it blew up for some reason! As per usual, `eleventyNavigation` got in a tizzy about nodes not being found.

Reply:
Ugh! I find this pretty confusing.

Reply:
Title of this page can now be set by layout, else defaults to `Posts tagged ‘[tag]’`.